support/modules/docker

```diff
@@ -27,11 +28,13 @@ case "$STATE" in
 
         >&2 echo "Stopping all running containers"
         [ -n "$(cat $prev_containers_file)" ] && docker stop $(cat $prev_containers_file)
+
+        [ -e ${run_args_file} ] && run_args=$(cat "${run_args_file}")
         for container in $(jq -r ".containers[]" "$FILES"/header/meta-data); do
             >&2 echo "Pulling new container [$container]"
             docker pull $container
-            >&2 echo "Starting new container [$container]"
-            docker run -dt $container
+            >&2 echo "Starting new container [$container] with args [${run_args}]"
+            docker run -dt ${run_args} $container
```

Comment:
Here you can simply do:

```bash
docker run -dt $(jq -r ".run_args | select (.!=null)") $container
```

(See [this discussion](https://github.com/stedolan/jq/issues/354) for why the "select" part is necessary)